frontend/src/components/project/rms/Overview.tsx

```diff
@@ -209,9 +220,117 @@ function RmsInfo({ rmsData }: { rmsData: RmsProject }) {
   );
 }
 
-export function Overview({ projectData }: { projectData: FmuProject }) {
-  const [isDialogOpen, setIsDialogOpen] = useState(false);
-  const rmsData = projectData.config.rms;
+function RmsProjectActions({
+  rmsData,
+  projectReadOnly: projectIsReadOnly,
+  setIsRmsProjectOpen,
+  isRmsProjectOpen,
+}: {
+  rmsData: RmsProject | null | undefined;
+  projectReadOnly: boolean | undefined;
+  setIsRmsProjectOpen: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
Suggest importing these from `React`, as that's how it is done in other places.

```suggestion
  setIsRmsProjectOpen: Dispatch<SetStateAction<boolean>>;
```

This suggestion also applies to `RmsEditorForm`.